pkg/deploy/deploy.go

```diff
@@ -292,7 +293,7 @@ func manageResource(ctx context.Context, cli client.Client, obj *unstructured.Un
 	// TODO: Remove this when we have generalize custom config requirements across all components
 	if componentName == "kserve" {
 		// do not reconcile kserve resource with annotation "opendatahub.io/managed: false"
-		if found.GetAnnotations()["opendatahub.io/managed"] == "false" {
+		if found.GetAnnotations()[annotation.ManagedAnnotation] == "false" {
```

Comment:
I would consider a more meaningful name, the suffix annotation is a bit redundant. For example `annotation.ManagedByODHOperator`

Reply:
One thing we could have also change as part of this PR is this bit https://github.com/opendatahub-io/opendatahub-operator/pull/955/files#r1565449423

Reply:
updated in https://github.com/opendatahub-io/opendatahub-operator/pull/967/commits/18095c6f9df686b3828b2ebb47297098a0c045f4